packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatAssistantBubble.tsx

```diff
@@ -567,6 +568,12 @@ export const AssistantBubble: FC<Props> = memo(
                                 <MantineIcon icon={IconBug} />
                             </ActionIcon>
                         )}
+
+                        <MessageModelIndicator
```

Comment:
I think if I'm on a thread, I don't need to see the model name on every message I get back. Why not a label or even the same component (model selector) next to the chat, but readOnly mode?

Reply:
My reasoning is, while not supported now, you could switch the models mid conversation